12/umbraco-commerce/reference/storefront-api/README.md

```diff
@@ -0,0 +1,611 @@
+* `Tax-Class` - The ID or Alias of the default tax class. If not supplied, will fallback to the default defined on the store.
+* `Billing-Country` - The ID or ISO Code of the default billing country. If not supplied, falls back to the default country defined on the store.
+* `Billing-Region` - The ID or ISO Code of the default billing region. 
+* `Shipping-Country` - The ID or ISO Code of the default shipping country. If not supplied, falls back to either the supplied billing country or default country defined on the store.
+* `Shipping-Region` - The ID or ISO Code of the default billing region.
+
+</details>
+
+<details>
+
+<summary>Expansion</summary>
+
+**Expansion** allows you to retrive additional data about related entities in the API output for a given resource.
+
+By default, where a resource is connected with another resource, such as an Order holding a reference to it's Currency, the Storefront API will return those connected resources as "Reference" objects which usually only contain the resources ID and Alias/Code. 
+
+```json
+{
+    "cartNumber": "CART-01280-054677-RP4L9",
+    "createDate": "2023-07-03T15:11:17.9220005",
+    "currency": {
+        "$type": "CurrencyRef",
+        "code": "GBP",
+        "id": "30b62176-6a9e-4a51-b2f0-7ce6c80a461a"
+    },
+    "customerInfo": { ... },
+    "discountCodes": [],
+    ...
+}
+```
+
+Should you wish to retrieve the connected resource, you could perform an additional get operation against that resources own endpoint, however this could result in multiple wasteful network requests. To help with this, you can pass an `expand` query parameter with a path to properties you wish to expand, which for those targeted properties will return the full resource object instead of the reference entity.
+
+**Request**
+
+```http
+GET /umbraco/commerce/storefront/api/v1/order/af697207-d370-4aee-824c-15711d43a9f2?expand=currency
+```
+
+**Response**
+
+```json
+{
+    "cartNumber": "CART-01280-054677-RP4L9",
+    "createDate": "2023-07-03T15:11:17.9220005",
+    "currency": {
+        "$type": "Currency",
+        "allowedCountries": [
+            {
+                "country": {
+                    "$type": "CountryRef",
+                    "code": "GB",
+                    "id": "af697207-d370-4aee-824c-15711d43a9f2"
+                }
+            }
+        ],
+        "code": "GBP",
+        "culture": "en-GB",
+        "id": "30b62176-6a9e-4a51-b2f0-7ce6c80a461a",
+        "name": "GBP"
+    },
+    "customerInfo": { ... },
+    "discountCodes": [],
+    ...
+}
+```
+
+The `expand` query parameter can accept a comma seperated list of property keys to expand for properties at the same level, and can also expand nested objects using a `[...]` syntax. In the following example we retrieve a Country entity, expanding it's `defaultCurrency` and `defaultPaymentMethod` whilst at the same time expanding the `allowedCountries.country` properties within the `defaultCurrency`.
```

Comment:
```suggestion
The `expand` query parameter can accept a comma-separated list of property keys to expand for properties at the same level. It can also expand nested objects using a `[...]` syntax. In the following example, we retrieve a Country entity. We will expand its `defaultCurrency` and `defaultPaymentMethod` whilst at the same time expand the `allowedCountries.country` properties within the `defaultCurrency`.
```